src/coreclr/src/vm/appdomainnative.cpp

```diff
@@ -19,33 +19,37 @@
 #include "clr/fs/path.h"
 using namespace clr::fs;
 
-FCIMPL3(Object*, AppDomainNative::CreateDynamicAssembly, AssemblyNameBaseObject* assemblyNameUNSAFE, StackCrawlMark* stackMark, INT32 access)
+// static
+void QCALLTYPE CreateDynamicAssembly(QCall::ObjectHandleOnStack assemblyName, QCall::StackCrawlMarkHandle stackMark, INT32 access, QCall::ObjectHandleOnStack retAssembly)
```

Comment:
You are missing `AppDomainNative::` that is making all builds to fail.

Reply:
Sorry. I tested with old dotnet/coreclr, but there are some mistakes when I apply it to dotnet/runtime by manually.